ios/Podfile.lock

```diff
@@ -1980,8 +1980,8 @@ SPEC CHECKSUMS:
   SDWebImageWebPCoder: af09429398d99d524cae2fe00f6f0f6e491ed102
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
   Turf: 13d1a92d969ca0311bbc26e8356cca178ce95da2
-  VisionCamera: 7d13aae043ffb38b224a0f725d1e23ca9c190fe7
-  Yoga: e64aa65de36c0832d04e8c7bd614396c77a80047
+  VisionCamera: fda554d8751e395effcc87749f8b7c198c1031be
+  Yoga: 13c8ef87792450193e117976337b8527b49e8c03
```

Comment:
no need to update Yoga as well

Reply:
This checksum depends on device or OS where pod is being installed.
i.e. 13c8ef87792450193e117976337b8527b49e8c03 on your device and e64aa65de36c0832d04e8c7bd614396c77a80047 on my device

If you check commit history, this line has changed many times depending on PR author's device where pod is installed and committed

Reply:
In that case what would you suggest?

Reply:
nothing to suggest as this is RN upstream issue
either value is fine to me

Reply:
But anyway I think in that case we can leave it as it is, and the PR is currently ready 😄